src/IB/IBFEInstrumentPanel.cpp

```diff
@@ -1110,8 +1110,11 @@ IBFEInstrumentPanel::getMeterRadius(const int meter_mesh_number)
 void
 IBFEInstrumentPanel::outputData(const double data_time)
 {
-    if (SAMRAI_MPI::getRank() == 0)
+    static const int mpi_root = 0;
+    if (SAMRAI_MPI::getRank() == mpi_root)
```

Comment:
It seems weird that we have the constant in one place and not the other. I would just revert this to be `== 0`.

Reply:
I have a slight preference for naming the constant in these cases, but can go either way.

Reply:
I deferred to Boyce's preference... sorry David :-).

Reply:
My first preference is consistency, so I will still call it a win :)